config/locales/en/general.yml

```
@@ -933,3 +934,25 @@ en:
   machine_learning:
     comments_summary: "Comments summary"
     info_text: "Content generated by AI / Machine Learning"
+  cookies_consent:
+    accept_all_cookies: "Accept all"
+    accept_essential_cookies: "Accept essential cookies"
+    message: "Cookies help us deliver our services. By using our services, you agree to our use of cookies."
+    more_info_link: "More information"
+    notice: "Your cookies preferences were saved."
+    setup: "Setup"
```

I think I've never seen the word "Setup" in this context :thinking:.

```suggestion
    manage: "Manage cookies"
```

Maybe we should  also rename the `SetupComponent` :thinking:.